cpp/tests/interop/to_arrow_device_test.cpp

```diff
@@ -711,6 +711,82 @@ TEST_F(ToArrowDeviceTest, StructColumn)
 template <typename T>
 using fp_wrapper = cudf::test::fixed_point_column_wrapper<T>;
 
+TEST_F(ToArrowDeviceTest, FixedPoint32Table)
```

Comment:
Copy-pasted the test body from `FixedPoing64Table` and modified types and expect_data.